javascript/javascript1/week3/activities.js

```diff
@@ -0,0 +1,51 @@
+const activities = [];
+function addActivity(date , activity , duration){
+activities.push( 
+    {
+        date : date,
+        activity : activity,
+        duration : duration,
+    }
+);
+}
```

Comment:
There's some indentation missing here:
```suggestion
function addActivity(date, activity, duration) {
    activities.push(  
        {
            date : date,
            activity : activity,
            duration : duration,
        }
    );
}
```

Reply:
There a shorthand for assigning objects when the property and variable name is the same:
```js
{ date, activity, duration } // this is the same as { date: date, activity: activity, duration: duration }
```

Reply:
Implemented. Thanks